docker/entrypoint.sh

```diff
@@ -10,7 +10,7 @@ migrations() {
     if [ "$BACKEND_ENABLED" == "True" ]; then
 
         python manage.py install_postgres_extensions
-        python manage.py migrate
+        python manage.py migrate --noinput
```

Comment:
Ah, TIL<sup>[source](https://docs.djangoproject.com/en/1.9/ref/django-admin/#django-admin-migrate)</sup> that

> **Changed in Django 1.9:**
> The `--no-input` alias was added.